.travis.yml

```diff
@@ -5,3 +5,9 @@ branches:
     - master
     - /^greenkeeper\/.*$/
 cache: yarn
+
+# Since the upgrade to Ubuntu Precise, yarn is not available to run scripts: `yarn run foo`
+# So let's install it manually
+before_install:
```

Comment:
Yes this is probably a Travis bug in the latest environment upgrade: https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming